.github/workflows/process_new_pull_request.yml

```diff
@@ -0,0 +1,112 @@
+name: Process New Pull Request
+
+on:
+  pull_request:
+    types: [opened]
+  merge_group:
+    types: [checks_requested]
+  issues:
```

Comment:
Ah, I'm sorry, I just realized that the oppiabot.yml trigger is for everything and not just the CLA verification.

Could you please have this workflow be only for the trigger when opening PRs (as described by the name), including the Oppiabot verification, and keep the other triggers in oppiabot.yml? Thanks.